### DISCUSSION TITLE
Is it possible for a bot to interact with a bot web app?

### DISCUSSION THREAD

**Discussion by GaloisField2718** [python-telegram-bot/python-telegram-bot#4176] [Q&A]
Hello everyone, 

I have a question for which I don't have a clear idea about how to do it. 
There is a bot 🤖 which is managing a webapp on telegram. I want to interact with a bot with it. 
How can I call functions into this webapp with my bot? Do you have some ideas about how to do it?

**Comment by Poolitzer** [comment]
Hi, you could call the web app like a normal URL, but it expects a working telegram JavaScript library, which you would need to reverse engineer how the apps provide them. And well, you need to somehow run the js things as well, so its not a simple get request

**Reply by GaloisField2718** [reply]
Ah yes ok i see. Thank you very much for your feedback.
